src/target/riscv_debug.c

```diff
@@ -157,6 +157,15 @@ static const riscv_csr_descriptor_s riscv_csrs[]={
 	{"mtval",	RV_CSR_MTVAL},
 	{"mip",	RV_CSR_MIP},
 };
+// fpu registers straight from gdb https://patchwork.kernel.org/project/qemu-devel/patch/20181228220731.4753-1-jimw@sifive.com/
+// actual order from gdb 13.2
+static const char *riscv_fpu_regs[] ={"t0","t1","t2", "t3","t4","t5","t6","t7",
+										"s0","s1","a0","a1","a2","a3","a4",
+										"a5","a6","a7","s2","s3","s4","s5",
+										"s6","s7","s8","s9","s10","s11","t8",
+										"t9","t10","t11"};
```

Comment:
The final string needs to be followed by a `,` to get clang-format to format this properly.